w3lib/form.py

```diff
@@ -1,65 +0,0 @@
-import warnings
-from io import BytesIO
-
-from w3lib.util import unicode_to_str
```

Comment:
would you mind adding deprecation warnings to these functions? I think this was the only place unicode_to_str is used, and the names are all wrong